content/en/monitors/monitor_status.md

```diff
@@ -78,7 +88,7 @@ For further investigation into your metrics evolution, use the [Metric Explorer]
 
```

Comment:
I can't make this as a suggestion since it's not a change you made, but hostname should be one word.